benchmarks/src/test/java/io/crate/action/job/NodeOperationCtxBenchmark.java

```diff
@@ -24,7 +24,7 @@
 
 import com.carrotsearch.hppc.cursors.IntCursor;
 import io.crate.executor.transport.NodeOperationTreeGenerator;
-import io.crate.operation.NodeOperation;
+import io.crate.execution.dsl.phases.NodeOperation;
```

Comment:
Do we have to encode `dsl` here? What differentiates this dsl from other dsl we have, e.g. LogicalPlan dsl. I'd leave out the dsl part.

Reply:
I'd like to move the actual execution classes in a different sub-package later on. So dsl will serve to distinguish between description and implementation